app/assets/stylesheets/govuk_publishing_components/components/_contextual-guidance.scss

```diff
@@ -15,11 +15,9 @@
 
 .gem-c-contextual-guidance__wrapper {
   position: relative;
-
-  @include govuk-font(19);
-
   margin-bottom: govuk-spacing(7);
 
+  @include govuk-font(19);
   @include govuk-media-query($from: tablet) {
```

Comment:
Nitpick: I think we should keep mixins visually associated with declarations rather than media queries (even though those are also mixins). So instead of:

```css
.gem-c-contextual-guidance__wrapper {
  position: relative;
  margin-bottom: govuk-spacing(7);

  @include govuk-font(19);
  @include govuk-media-query($from: tablet) {
```

...I'd vote for:

```css
.gem-c-contextual-guidance__wrapper {
  position: relative;
  margin-bottom: govuk-spacing(7);
  @include govuk-font(19);

  @include govuk-media-query($from: tablet) {
```

...but it's only a minor thing.